packages/mdc-ripple/util.js

```diff
@@ -34,7 +34,16 @@ export function supportsCssVariables(windowObj) {
     windowObj.CSS.supports('(--css-vars: yes)') &&
     windowObj.CSS.supports('color', '#00000000')
   );
-  return explicitlySupportsCssVars || weAreFeatureDetectingSafari10plus;
+  // See: https://caniuse.com/#search=custom%20properties
+  // See: README section on Edge
+  const weAreFeatureDetectingEdge = (
```

Comment:
But won't it make it less clear that our goal is FD Edge? So when we change the FD feature, we are not always changing the function name?